apps/prairielearn/src/components/Scorebar.html.ts

```diff
@@ -0,0 +1,18 @@
+import { html } from '@prairielearn/html';
+
+export function Scorebar(
+  score: number | null,
+  { minWidth = '5em', maxWidth = '20em' }: { minWidth?: string; maxWidth?: string } = {},
```

Comment:
Conventionally, there would be a single object here with three properties.

Reply:
I considered that, but given that the vast majority of cases to call this component will have the score as the only parameter, I followed the convention used in `formatDate` and other similar functions. I believe there is no confusion in assuming that, if a single parameter is passed to this component, it is the score, and the parameters that are not the score are additional options. That said, if you have a strong objection to it I can change it.